README.md

```diff
@@ -31,6 +31,7 @@ use the form `$(variable['subkey'])`. Special values: `$(tag)` references the fu
 * `log_key`: By default, the whole log record will be sent to CloudWatch. If you specify a key name with this option, then only the value of that key will be sent to CloudWatch. For example, if you are using the Fluentd Docker log driver, you can specify `log_key log` and only the log message will be sent to CloudWatch.
 * `log_format`: An optional parameter that can be used to tell CloudWatch the format of the data. A value of `json/emf` enables CloudWatch to extract custom metrics embedded in a JSON payload. See the [Embedded Metric Format](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html).
 * `role_arn`: ARN of an IAM role to assume (for cross account access).
+* `auto_create_group`: Automatically create log groups (and add tags). Valid values are "true" or "false" (case insensitive). Defaults to true.
```

Comment:
it used to default to false- I think we should keep that, otherwise folks will complain about this being a backwards breaking change

Reply:
There is a short discussion about it in #96. We can go with `false`. Either way works for _me_ haha.

Reply:
something weird happened. I got a log stream named `$(tag[0]).$(ident)syslog.0` verbatim.